docs/deployment/installation.md

```diff
@@ -214,28 +215,39 @@ $ tar xvfz opencti-release-{RELEASE_VERSION}.tar.gz
 The main application has just one JSON configuration file to change and a few Python modules to install
 
 ```bash
-$ cd opencti
-$ cp config/default.json config/production.json
+cd opencti
+cp config/default.json config/production.json
 ```
 
 Change the *config/production.json* file according to your configuration of ElasticSearch, Redis, RabbitMQ and S3 bucket as well as default credentials (the `ADMIN_TOKEN` must be a [valid UUID](https://www.uuidgenerator.net/)).
 
 #### Install the Python modules
 
 ```bash
-$ cd src/python
-$ pip3 install -r requirements.txt
-$ cd ../..
+cd src/python
+pip3 install -r requirements.txt
+cd ../..
 ```
 
 #### Start the application
 
 The application is just a NodeJS process, the creation of the database schema and the migration will be done at starting.
 
+> Please verify that yarn version is greater than 4 and node version is greater than v19.
```

Comment:
strictly greater or greater or equal ?

Reply:
On my tests greater or equals, I'm adding it thanks.